objects/otestplayer/Create_0.gml

```diff
@@ -0,0 +1,5 @@
+moveSpd = 1.5;
```

Comment:
Again, this violates the "no abbreviations" rule in the style guide. Rename these using the full word